unittests/test_cli.py

```diff
@@ -591,6 +591,42 @@ def test_unload_module(run_reframe, user_exec_ctx):
     assert returncode == 0
 
 
+def test_unuse_module_path(run_reframe, user_exec_ctx, monkeypatch):
+    ms = rt.runtime().modules_system
+    if ms.name == 'nomod':
+        pytest.skip('no modules system found')
+
+    # We need to have MODULEPATH set to unuse a module
```

Comment:
This comment is a bit confusing. What do you mean?

Reply:
I think in order to test the unuse, you should add a path to the `MODULEPATH`, then invoke reframe with the option to unuse it and then check that the expected warning was printed in the output.